modules/Book.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable */
```

Comment:
- Please remove all disabled linter from your project, remember linters help to ensure that we write code that is clean and easy to maintain. Please fix this 😄

Reply:
Okay, I would delete codes that disable linters. Thanks for the review